{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 5, "total_error": 0}, "tests": {"0": {"status": "FAIL", "message": "           AttributeError: 'RootLogger' object has no attribute 'fail'\n", "test_name": "test_mapview_widget_kpiStatistics[requestDict0-expectedDict0]", "rerun": "0"}, "1": {"status": "FAIL", "message": "           AttributeError: 'RootLogger' object has no attribute 'fail'\n", "test_name": "test_mapview_widget_kpiStatistics[requestDict1-expectedDict1]", "rerun": "0"}, "2": {"status": "FAIL", "message": "       AssertionError: soft assertion failures:\n       1. [P50 validation] Expected <-104.0> to be equal to <0>, but was not.\n       2. [AvgUserCount validation] Expected <1834.0> to be equal to <0>, but was not.\n", "test_name": "test_mapview_widget_kpiStatistics[requestDict2-expectedDict2]", "rerun": "0"}, "3": {"status": "FAIL", "message": "           AttributeError: 'RootLogger' object has no attribute 'fail'\n", "test_name": "test_mapview_widget_kpiStatistics[requestDict3-expectedDict3]", "rerun": "0"}, "4": {"status": "FAIL", "message": "       AssertionError: soft assertion failures:\n       1. [P10 validation] Expected <0.039> to be equal to <0>, but was not.\n", "test_name": "test_mapview_widget_kpiStatistics[requestDict4-expectedDict4]", "rerun": "0"}}, "suite_name": "test_mapview.py"}}}, "date": "January 21, 2022", "start_time": 1642762782.4712508, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "0", "fail": "5", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "5"}